terraform/deployments/tfc-configuration/datagovuk-infrastructure.tf

```diff
@@ -0,0 +1,93 @@
+module "datagovuk-infrastructure-integration" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.9.0"
+
+  organization      = var.organization
+  workspace_name    = "datagovuk-infrastructure-integration"
+  workspace_desc    = "The datagovuk-infrastructure module is responsible for the AWS resources which constitute the EKS cluster."
+  workspace_tags    = ["integration", "datagovuk-infrastructure", "eks", "aws"]
+  terraform_version = "1.7.0"
+  execution_mode    = "remote"
+  working_directory = "/terraform/deployments/datagovuk-infrastructure/"
+  trigger_patterns  = ["/terraform/deployments/datagovuk-infrastructure/**/*"]
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "samsimpson1/govuk-aws"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production" = "write"
+    "GOV.UK Production"     = "write"
+  }
+
+  variable_set_names = [
+    "aws-credentials-integration",
+    "common",
+    "common-integration"
+  ]
+}
+
+module "datagovuk-infrastructure-staging" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.9.0"
+
+  organization      = var.organization
+  workspace_name    = "datagovuk-infrastructure-staging"
+  workspace_desc    = "The datagovuk-infrastructure module is responsible for the AWS resources which constitute the EKS cluster."
+  workspace_tags    = ["staging", "datagovuk-infrastructure", "eks", "aws"]
+  terraform_version = "1.7.0"
+  execution_mode    = "remote"
+  working_directory = "/terraform/deployments/datagovuk-infrastructure/"
+  trigger_patterns  = ["/terraform/deployments/datagovuk-infrastructure/**/*"]
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "samsimpson1/govuk-aws"
```

Comment:
Just checking these branch refs are for cleanup later right?

Reply:
Yep, it'll be set to main once its in working order